app/controllers/application_controller.rb

```diff
@@ -205,13 +203,8 @@ def invalid_auth_token(_exception)
   end
 
   def user_fully_authenticated?
-    !reauthn? && user_signed_in? &&
-      two_factor_enabled? &&
-      session['warden.user.user.session'] &&
-      !session['warden.user.user.session'].try(
-        :[],
-        TwoFactorAuthenticatable::NEED_AUTHENTICATION,
-      )
+    !reauthn? && user_signed_in? && two_factor_enabled? && session['warden.user.user.session'] &&
```

Comment:
I don't feel as strongly about this one, but it would be nice to have these on separate lines still, bummer there's no config for it

Reply:
I feel a bit more strongly that these should be on separate lines (including the orig line 208). This is a complicated condition and going line by line is easier to parse. The last condition does just fit on a 100 char line width, so that's better.